demisto_sdk/commands/common/errors.py

```diff
@@ -467,6 +469,12 @@ def invalid_integration_parameters_display_name(invalid_display_names):
                "Integration parameters display name should be capitalized and spaced using whitespaces " \
                "and not underscores ( _ )."
 
+    @staticmethod
+    @error_code_decorator
+    def is_valid_integration_file_path(integration_file):
```

Comment:
This error is good for packaged integrations but we should create a different one for unified integrations (`integration-something.yml`) cause we still have those.

Reply:
done